runing/repositories.py

```
@@ -0,0 +1,54 @@
+from sqlalchemy.orm import Session
+from typing import List, Optional
+import models
+import schemas
+
+class UserRepository:
+    def __init__(slef, db: Session):
+        self.db = db 
+        
+    def get_user(self, user_id: int) -> Optional[models.User]:
+        return self.db.query(models.User).filter(modles.User.email == email).first()
```

Ruff (0.11.9)

Undefined name `modles`

(F821)